components/channel_header_dropdown/menu_items/close_channel/close_channel.test.tsx

```diff
@@ -17,7 +17,7 @@ describe('components/ChannelHeaderDropdown/MenuItem.CloseChannel', () => {
     };
 
     it('should match snapshot', () => {
-        const wrapper = shallow(<CloseChannel {...baseProps}/>);
+        const wrapper: ShallowWrapper<any, any, CloseChannel> = shallow(<CloseChannel {...baseProps}/>);
```

Comment:
You can simplify these a bit by moving the type parameter to `shallow` instead of putting it on the wrapper itself
```suggestion
        const wrapper = shallow<CloseChannel>(<CloseChannel {...baseProps}/>);
```

Reply:
Hi @hmhealey , all those suggestions are ready!